person.rb

```diff
@@ -23,4 +25,8 @@ def of_age?
   def can_use_service?
     of_age? || @parent_permission
   end
+
+  def add_rental(rental)
+    rentals << rental
```

Comment:
`add_rental` should take the book and date as a parameter. A new rental should be initialized using the keyword `new` and the `person` itself should be sent as a parameter to create the new rental achieving the has-many association. Like this: `Rental.new(date, book, self)`. Kindly update your code accordingly.